suse_migration_services/units/prepare.py

```diff
@@ -92,6 +92,7 @@ def main():
                 Path.create(trust_anchor)
                 for cert in certificates:
                     cert_file = os.sep.join([root_trust_anchor, cert])
+                    print(cert_file)
```

Comment:
do we want to show the cert ?
I think we should remove this line

Reply:
sorry left that in during debugging a test failure. i will remove it. thanks for catching